### PR TITLE
Featchersnisha

### PR DESCRIPTION
**Pull request by nishaeazyerp**

### CODE REVIEW

File: controllers/GlobalController.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  // --------------------------GetUserRoleList -----------------------------------//
+
+  static async getUserRoleList(req, res) {
+    try {
+      //  const formId = req.params;
+      //   console.log(formId);
+      const tokenDetails = req.user
+      console.log('1111111111', req.user)
```
Comment (GarvishJainErp) [inline controllers/GlobalController.js:43]: remove the logs

File: services/GlobalService.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  static async getUserList() {
+    const objResultSet = {
+      status: false,
+      statuscode: 417,
+      message: '',
+      dtList: null,
+    }
+    try {
+      let sQuery = `
+                SELECT DISTINCT * FROM tbl_usermaster WHERE usertype <> '10' AND isactive = TRUE
```
Comment (GarvishJainErp) [inline services/GlobalService.js:24]: written query in single line in the tild sign

File: services/GlobalService.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            `
+      writeLog('-----GetUserList Query-----', sQuery)
+
+      const dtUserList = await postgreConnection.query(sQuery) // Assuming you have a function to execute queries
+      console.log('dtUserList', dtUserList)
+      ;(objResultSet.statuscode = 200),
```
Comment (GarvishJainErp) [inline services/GlobalService.js:30]: remove the semi-colon and also the brackets

File: services/GlobalService.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    const objResultSet = {
+      status: false,
+      statuscode: 417,
+      message: '',
+      dtList: null,
+    }
+
+    try {
+      const sQuery = `
+          SELECT roleid,
```
Comment (GarvishJainErp) [inline services/GlobalService.js:58]: written query in a single line

File: services/GlobalService.js [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+          FROM tbl_secuserrole
+          WHERE userid = ${tokenDetails.UserId} AND menuid = ${FormId};
+        `
+
+      writeLog('-----GetUserRoleList Query-----  ' + sQuery)
+
+      const dtAllModuleList = await postgreConnection.query(sQuery)
+      console.log('dtAllModuleList', dtAllModuleList)
+
+      ;(objResultSet.statuscode = 200),
```
Comment (GarvishJainErp) [inline services/GlobalService.js:73]: same remove the semi-colon and brackets and logs

File: services/GlobalService.js [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+      writeLog('-----Get tbl_generate_newcode Data Query-----  ' + sQuery)
+
+      let dt = await postgreConnection.query(sQuery)
+      console.log('2222222222', dt)
+      //    const row = dt.rows[0]
+      console.log(dt[0].columnname)
+
+      if (dt.length > 0) {
+        console.log('Result or rows is empty or undefined')
+        sQuery = `
```
Comment (GarvishJainErp) [inline services/GlobalService.js:114]: query written a single line

Comment (nishaeazyerp) [reply services/GlobalService.js]: resolved

File: services/GlobalService.js (hunk 3 above)
Comment (nishaeazyerp) [reply services/GlobalService.js]: resolved

File: services/GlobalService.js (hunk 4 above)
Comment (nishaeazyerp) [reply services/GlobalService.js:57]: resolved

File: services/GlobalService.js (hunk 5 above)
Comment (nishaeazyerp) [reply services/GlobalService.js]: resolved

**Review by GarvishJainErp — APPROVED** [review]
ok

**Pull request closed without merging**